src/routes/sse.ts

```diff
@@ -0,0 +1,123 @@
+import { type Request, type Response, Router } from "express"
+import { instance as createLogger } from "../utils/logger"
+import { validateBroadcastBody, validateSendRequest } from "../validation/sse"
+
+const router = Router()
+const logger = createLogger("routes.sse")
+
+const connections = new Map<string, Response>()
+const HEARTBEAT_INTERVAL_MS = 30000 //30seg
+
+
+const generateConnectionId = (): string => {
+  return `conn_${Date.now()}_${Math.random().toString(36).substr(2, 9)}`
+}
+
+
+router.get("/sse/connect", (req: Request, res: Response) => {
```

Comment:
Separate the router from the business logic.
In the router we will keep only the routing. Set the path and the middlewares and functions that need to be called.

something like:

```
import { validate } from "../validation/sse"
import { function, anotherfunction } from "../controllers/sse"

router.get("/connect", validate, function)
router.post("/broadcast", validate, anotherfunction)

export default router
```